src/app/view/components/_common/searchable-dropdown/component/searchable-dropdown.component.ts

```diff
@@ -129,7 +158,11 @@ export class SearchableDropdownComponent extends BaseComponent implements Contro
     private handleClick(item: any): void {
         if (this.value !== item) {
             this.value = item;
-            this.valueString = item[this.labelPropertyName];
+            if (typeof this.labelPropertyName === 'object') {
```

Comment:
There is a better way to know if is an array: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray